packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.scss

```diff
@@ -7,12 +7,6 @@
 		}
 
 		.checkbox {
-			&:after {
-				position: static;
-				display: inline;
-				content: '*';
-			}
-
```

Comment:
Don’t change a rule globally if the goal is to introduce a specific rule.
The idea is good: remove the star, but only in the ListView
- revert the css removal in FieldTemplate
- add a specific css in ListView style to override the global rule

Reply:
Drop this suggestion after @frassinier's rework on style